cfn-guard-rulegen-lambda/README.md

```diff
@@ -13,6 +13,7 @@
 * [Rust](https://rustup.rs/) (See the installation instructions in the [top-level README](../README.md#install-rust))
 * If building on a Mac, you'll need [Homebrew](https://brew.sh/).  
 * If building on Ubuntu, you'll need to run `sudo apt-get update; sudo apt install build-essential` if you haven't already
+* If leveraging, cloud infrastructure in code and provisioning it through AWS CloudFormation, you'll need to install or update the [AWS CDK CLI] from npm (requires [Node.js ≥ 10.13.0](https://nodejs.org/download/release/latest-v10.x/)): `npm i -g aws-cdk`
```

Comment:
```suggestion
* If provisioning with the Cloud Development Kit (CDK), you'll need to install or update the [AWS CDK Toolkit](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) from npm (requires [Node.js ≥ 10.13.0](https://nodejs.org/download/release/latest-v10.x/)): `npm i -g aws-cdk`
```

Reply:
Ultimately, its AWS CloudFormation, which provisions the infrastructure, and CDK is being used to leverage cloud infrastructure in code. Please see: https://github.com/aws/aws-cdk, which has this description.